### DISCUSSION TITLE
Which is your fav Octocat?

### DISCUSSION THREAD

**Discussion by mvkaran** [mvkaran/monacloud#3] [Show and tell]
The [Octodex](https://octodex.github.com/) has hundreds of official Octocats. Here are some of my favourites.

Which one do you like? Upvote! 😄

**Comment by mvkaran** [comment]
### [Ironcat](https://octodex.github.com/ironcat/)

<img src="https://octodex.github.com/images/ironcat.jpg" height="300" alt="Ironcat">

**Reply by lordvlads77** [reply]
IronCat for the win hehe

**Comment by mvkaran** [comment]
### [Hulatocat](https://octodex.github.com/hulatocat/)

<img src="https://octodex.github.com/images/hula_loop_octodex03.gif" height="300" alt="Hulatocat">

**Reply by bagatka** [reply]
So cute :3

**Comment by mvkaran** [comment]
### [Octobi Wan Catnobi](https://octodex.github.com/octobiwan/)

<img src="https://octodex.github.com/images/octobiwan.jpg" height="300" alt="Octobi Wan Catnobi">

**Comment by mvkaran** [comment]
### [Baracktocat](https://octodex.github.com/baracktocat/)

<img src="https://octodex.github.com/images/baracktocat.jpg" height="300" alt="Baracktocat">

**Comment by mvkaran** [comment]
### [Daftpunktocat-Guy](https://octodex.github.com/daftpunktocat-guy/)

<img src="https://octodex.github.com/images/daftpunktocat-guy.gif" height="300" alt="Daftpunktocat-Guy">

**Reply by CodingIsLove** [reply]
Daft Punk Vibes 🚀

**Reply by bagatka** [reply]
WiN!

**Reply by IlmLV** [reply]
😎

**Comment by mvkaran** [comment]
### [Mona Lovelace](https://octodex.github.com/mona-lovelace/)

<img src="https://octodex.github.com/images/mona-lovelace.jpg" height="300" alt="Mona Lovelace">

**Comment by mvkaran** [comment]
### [Nuxtocat](https://octodex.github.com/nuxtocat/)

<img src="https://user-images.githubusercontent.com/8726608/139397303-83a84013-620e-4873-9535-5789c0caaddf.gif" height="300" />
